BBKoffieTuin/Assets/Scripts/Quiz/QuizCompleteAudioPlayer.cs

```diff
@@ -0,0 +1,42 @@
+using System.Linq;
+using Audio;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Quiz
+{
+    public class QuizCompleteAudioPlayer : MonoBehaviour
+    {
+        [System.Serializable]
+        public struct AnswerClip
+        {
+            public int AnswersCorrect;
+            public AudioClip clip;
+        }
```

Comment:
Persoonlijk vind ik het nice als je hiet ook een losse class van maakt dan kan het vaker gebruikt worden en makkelijker te vinden. not sure of we hier ook echt een conventie voor hebben